packages/genesys-spark-components/src/components/stable/gux-tag/example.html

```diff
@@ -36,6 +36,42 @@ <h2>Disabled</h2>
   <gux-tag disabled accent="invalid">Invalid</gux-tag>
 </div>
 
+<h2>Size</h2>
+<div class="tags">
+  <gux-tag size="small">Small</gux-tag>
+  <gux-tag size="large">Large</gux-tag>
+</div>
+
+<h2>Emphasis</h2>
```

Comment:
We also have an `Active` section in the example file which are basically the same tags as the `bold` under the emphasis section. I am not sure how you will differentiate the two or maybe its fine.

Reply:
@gavin-everett-genesys yes, it's confusing. I've updated the docs to me more clear.